deeppavlov/models/multitask_pal_bert/multitask_pal_bert.py

```diff
@@ -0,0 +1,468 @@
+
+        if self.multilabel and not self.one_hot_labels:
+            raise RuntimeError(
+                "Use one-hot encoded labels for multilabel classification!"
+            )
+
+        if self.multilabel and not self.return_probas:
+            raise RuntimeError(
+                "Set return_probas to True for multilabel classification!"
+            )
+
+        if self.gradient_accumulation_steps > 1 and not self.steps_per_epoch:
+            raise RuntimeError(
+                "Provide steps per epoch when using gradient accumulation"
+            )
+
+        super().__init__(
+            optimizer_parameters=self.optimizer_parameters,
+            lr_scheduler=self.lr_scheduler_name,
+            lr_scheduler_parameters=self.lr_scheduler_parameters,
+            **kwargs,
+        )
+
+    @overrides
+    def init_from_opt(self) -> None:
+        """Initialize from scratch `self.model` with the architecture built
+        in `model_func (MultitaskBert)` method of this class along with
+        `self.optimizer` as `self.optimizer_name` from `torch.optim` and
+        parameters `self.optimizer_parameters`, optionally initialize
+        `self.lr_scheduler` as `self.lr_scheduler_name` from
+        `torch.optim.lr_scheduler` and parameters `self.lr_scheduler_parameters`
+        """
+        if self.config and os.path.exists(self.config):
+            self.bert_config = BertConfig.from_json_file(self.config)
+            self.bert_config.num_tasks = len(self.task_names)
+            self.model = BertForMultiTask(
+                self.bert_config, self.tasks_num_classes)
+            self.model.to(self.device)
+        else:
+            raise ValueError("Config File does not exist at", self.config)
+
+        if self.pretrained_bert:
+            partial = torch.load(self.pretrained_bert, map_location="cpu")
+            model_dict = self.model.bert.state_dict()
+            update = {}
+            for n, p in model_dict.items():
+                if "aug" in n or "mult" in n:
+                    update[n] = p
+                    if "pooler.mult" in n and "bias" in n:
+                        update[n] = partial["pooler.dense.bias"]
+                    if "pooler.mult" in n and "weight" in n:
+                        update[n] = partial["pooler.dense.weight"]
+                else:
+                    for val in [n, "bert." + n, "cls." + n]:
+                        if val in partial:
+                            update[n] = partial[val]
+            self.model.bert.load_state_dict(update)
+            log.info("Bert Model Weights Loaded.")
+        else:
+            raise ConfigError("No pre-trained BERT model is given.")
+
+        no_decay = ["bias", "gamma", "beta"]
+        base = ["attn"]
+        model_parameters = [
+            {
+                "params": [
+                    p
+                    for n, p in self.model.named_parameters()
+                    if not any(nd in n for nd in no_decay)
```

Comment:
I think it's better for readability to define them as two lambda functions

Reply:
(this string and next one)

Reply:
Replaced with two lambda functions.